src/index.html

```diff
@@ -0,0 +1,299 @@
+        Get Free trial
+      </button>
+    </div>
+    <div class="customer__item customer__item--right">
+      <img src="../img/Image.png" />
+    </div>
+  </section>
+  <section class="companies">
+    <img src="../img/logo1.PNG" class="companies__item" />
+    <img src="../img/logo2.PNG" class="companies__item" />
+    <img src="../img/logo3.PNG" class="companies__item" />
+    <img src="../img/logo4.PNG" class="companies__item" />
+    <img src="../img/logo3.PNG" class="companies__item" />
+  </section>
+  <section class="sales">
+    <div class="sales__item">
+      <p
+        class="sales__title sales__title--small">
+        Discover More</p>
+      <p class="sales__title">Analyze your sales and marketing leads
+      </p>
+      <p class="sales__text">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+        nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua invidunt ut labore.</p>
+    </div>
+    <div class="sales__item">
+      <div class="sales__details">
+        <span class="sales__icon"><img
+            src="../img/icons.png" /></span>
+        <p class="sales__subtitle">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          <br/>eiusmod.</p>
+      </div>
+      <div class="sales__details">
+        <span class="sales__icon"><img
+            src="../img/icons.png" /></span>
+        <p class="sales__subtitle">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          <br/>eiusmod.</p>
+      </div>
+      <div class="sales__details">
+        <span class="sales__icon"><img
+            src="../img/icons.png" /></span>
+        <p class="sales__subtitle">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          <br/>eiusmod.</p>
+      </div>
+    </div>
+  </section>
+  <section class="features">
+    <p class="features__title">
+      Features
+    </p>
+    <div
+      class="features__card">
+      <div class="features__item">
+        <img src="../img/3.png" class="features__img" />
+        <p class="features__header">High security to protect from piracy</p>
+        <p class="features__text">Lorem ipsum dolor sit amet, consetetur sadipscing
+          elitr, sed diam nonumy.</p>
+      </div>
+      <div class="features__item features__item--outline">
+        <img src="../img/3.png" class="features__img" />
+        <p class="features__header">High security to protect from piracy</p>
+        <p class="features__text">Lorem ipsum dolor sit amet, consetetur sadipscing
+          elitr, sed diam nonumy.</p>
+      </div>
+      <div class="features__item features__item--outline">
+        <img src="../img/3.png" class="features__img" />
+        <p class="features__header">High security to protect from piracy</p>
+        <p class="features__text">Lorem ipsum dolor sit amet, consetetur sadipscing
+          elitr, sed diam nonumy.</p>
+      </div>
+    </div>
+  </section>
+  <section class="stuff">
```

Comment:
`stuff` is too broad. Maybe something like `testimonials` would be specific. Should you decide to change the name, please change everywhere you have `stuff` in this block.